tools/lnx-cibuild.sh

```diff
@@ -31,12 +31,16 @@ if [ "${ENABLE_STYLE_TESTING}" = "On" ]; then
   _build_name=${_build_name}"-coding-style"
 fi
 
-# add unity to build name if requested
-if [ "${ENABLE_UNITYBUILD}" = "On" ]; then
-  _build_name=${_build_name}"-unity-build"
+# add class-testing to build name if requested
+if [ "${ENABLE_CLASS_TESTING}" = "On" ]; then
+  _build_name=${_build_name}"-testClass"
```

Comment:
just being picky, why not follow the style of the other build names and make it `test-class`? rest looks good. +1 for merge